plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -256,13 +255,18 @@ public synchronized void setTableProperties(ConnectorSession session, ConnectorT
         SchemaTableName tableName = handle.schemaTableName();
 
         TableInfo oldInfo = tables.get(tableName);
-        Map<String, Object> newProperties = Stream.concat(
-                        oldInfo.properties().entrySet().stream()
-                                .filter(entry -> !properties.containsKey(entry.getKey())),
-                        properties.entrySet().stream()
-                                .filter(entry -> entry.getValue().isPresent()))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
Properties should already have correct types - would it be enough to add a `get()` here? Like:
```java
.collect(Collectors.toMap(Map.Entry::getKey, entry -> entry.getValue().get()));
```